File: Level-0/59.java
class Solution {
    public int solution(int[] sides) {
        int answer = 0;
        int n = Math.abs(sides[1] - sides[0]);
        int m = sides[0] + sides[1];
        for(int i = n + 1; i < m; i++){
            answer++;
        }
        return answer;
    }
}

File: Level-0/59.js
function solution(sides) {
    const a = Math.abs(sides[0] - sides[1]);
    const b = sides[0] + sides[1];
    
    let answer = 0;
    for(let i = a + 1; i < b; i++) {
        answer++;
    }
    
    return answer;
}
